models/public/retinanet-tf/retinanet-tf.md

```diff
@@ -0,0 +1,97 @@
+# retinanet-tf
+
+## Use Case and High-Level Description
+
+RetinaNet is the dense object detection model with ResNet50 backbone, originally trained on Keras\*, then
+converted to TensorFlow\* protobuf format. For details, see [paper](https://arxiv.org/abs/1708.02002),
+[repository](https://github.com/fizyr/keras-retinanet).
+
+### Steps to Reproduce Keras\* to TensorFlow\* Conversion
+
+1. Clone the original [repository](https://github.com/fizyr/keras-retinanet)(tested on `47fdf189` commit)
+2. Download the original model from [here](https://github.com/fizyr/keras-retinanet/releases/download/0.5.1/resnet50_coco_best_v2.1.0.h5)
+4. Get conversion script:
+
+    1. Get conversion script from [repository](https://github.com/amir-abdi/keras_to_tensorflow):
+        ```sh
+        git clone https://github.com/amir-abdi/keras_to_tensorflow.git
+        ```
+    1. (Optional) Checkout the commit that the conversion was tested on:
+        ```
+        git checkout c841508a88faa5aa1ffc7a4947c3809ea4ec1228
+        ```
+    1. Apply `keras_to_tensorflow.patch`:
+        ```
+        git apply keras_to_tensorflow.patch
+        ```
+    1. Run script:
+        ```
+        python keras_to_tensorflow.py --input_model=<model_in>.h5 --output_model=<model_out>.pb
+        ```
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Object detection                          |
+| GFlops                          | 238.9469                                  |
+| MParams                         | 64.9706                                   |
+| Source framework                | TensorFlow\*                              |
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name: `input_1`, shape: [1x1333x1333x3], format: [BxHxWxC],
+   where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order: BGR.
```

Comment:
Unnecessary indent?

Reply:
The list is also overly indented (check the GitHub preview, you'll see that it's parsed as a code block).